dask/dataframe/groupby.py

```diff
@@ -895,8 +895,14 @@ def _groupby_apply_funcs(df, *index, **kwargs):
 
 
 def _compute_sum_of_squares(grouped, column):
-    base = grouped[column] if column is not None else grouped
-    return base.apply(lambda x: (x ** 2).sum())
+    # Note: CuDF cannot use `groupby.apply`.
+    # Need to unpack groupby to compute sum of squares
+    if hasattr(grouped, "grouper"):
+        keys = grouped.grouper
+    else:
+        keys = grouped.grouping._key_columns
```

Comment:
Is this a cudf thing? I don't see `_key_columns` in pandas master or dask.

Might want to include a note there, so that it isn't removed accidentally.

Reply:
Ah good point.  `_key_columns` is indeed cudf-specific, so I added a note